src/shared/components/FormComponents/TagsController/TagsController.tsx

```diff
@@ -71,19 +72,14 @@ export const TagsController = <T extends FieldValues>({
               error={!!error || providedError}
               helperText={error?.message || helperText}
               value={value}
-              showInputLabel={showInputLabel}
```

Comment:
It seems this will cause braking the layout, if there are more than two rows:
before:
![image](https://github.com/user-attachments/assets/7f96f640-f006-4276-b142-5d88abc5302f)
after:
![image](https://github.com/user-attachments/assets/c22e67bb-18db-4afa-8c29-2417d7d3b993)